settings.gradle

```diff
@@ -28,8 +28,8 @@ if (!System.getenv().containsKey("SUB_BUILD")) {
 } else {
```

Comment:
The comment just above here is out of date now. Add `ALL_CONNECTORS` to the list. It's minor but I think just `CONNECTORS` would be fine too and maybe more intuitive.

Reply:
I use `ALL_CONNECTORS` to distinguish it from `CONNECTORS_BASE`.